### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by mvieth**

### CODE REVIEW

**Comment by mvieth** [comment]
BTW will there be API docs for older versions again? I kind of miss them ...

**Comment by kunaltyagi** [comment]
> BTW will there be API docs for older versions 

Docs for HEAD@master are created via CI. We don't have tooling for supporting versioned docs. Easiest would be: a PR to convert the docs from doxygen to sphinx and then we use RTD to manage the versions

**Comment by mvieth** [comment]
Previously, the docs generated by doxygen seemed to be stored in folders for the versions, e.g. `http://docs.pointclouds.org/1.9.1/group__common.html` for 1.9.1. Maybe that can be revived? And there could be a link for each version in a drop-down on https://pointclouds.org/ or something?

File: sample_consensus/include/pcl/sample_consensus/rransac.h [hunk 1]
```diff
@@ -70,7 +74,7 @@ namespace pcl
       using SampleConsensus<PointT>::inliers_;
       using SampleConsensus<PointT>::probability_;
 
-      /** \brief RANSAC (Randomized RAndom SAmple Consensus) main constructor
+      /** \brief RANSAC (Randomized RANdom SAmple Consensus) main constructor
```
Comment (kunaltyagi) [inline sample_consensus/include/pcl/sample_consensus/rransac.h:77]: The acronyms don't match with the expansion

Comment (mvieth) [reply sample_consensus/include/pcl/sample_consensus/rransac.h]: Oh, good eye! I only noticed the "n", haha

File: sample_consensus/include/pcl/sample_consensus/rransac.h [hunk 2]
```diff
@@ -81,7 +85,7 @@ namespace pcl
         max_iterations_ = 10000;
       }
 
-      /** \brief RRANSAC (RAndom SAmple Consensus) main constructor
+      /** \brief RRANSAC (RANdom SAmple Consensus) main constructor
```
Comment (kunaltyagi) [inline sample_consensus/include/pcl/sample_consensus/rransac.h:88]: Same error here

Comment (mvieth) [reply sample_consensus/include/pcl/sample_consensus/rransac.h]: fixed

**Review by larshg — APPROVED** [review]

**Pull request merged**